packages/financial-health-calculator/src/helpers/number-formatter.ts

```diff
@@ -0,0 +1,7 @@
+export const formatCurrency = (number: number, currency: string) => {
+  return new Intl.NumberFormat('en-GB', { style: 'currency', currency }).format(number);
```

Comment:
Try to get the user's preferred locale via the Accept-Language header

Reply:
Resolved in ec2df43d7ab634ed0eb49fd485433e0b3ef7d46b